sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/Crashes.java

```diff
@@ -287,24 +287,19 @@ private synchronized boolean hasInstanceCrashedInLastSession() {
      * Implements {@link #getLastSessionCrashReport(LastCrashErrorReportListener)} at instance level.
      */
     private synchronized void getInstanceLastSessionCrashReport(LastCrashErrorReportListener listener) {
-        switch (mLastSessionCrashProcessingStatus) {
-            case PREPARE_PROCESSING:
-            case PROCESSING:
-                if (mLastCrashErrorReportListeners == null) {
-                    mLastCrashErrorReportListeners = new ArrayList<>();
-                }
-                mLastCrashErrorReportListeners.add(listener);
-                break;
-            case PROCESSED:
-                listener.onSuccess(mLastSessionErrorReport);
-                break;
-            case PROCESSING_FAILED:
-                listener.onFailure();
-                break;
-            case NOT_FOUND:
```

Comment:
That didn't help for 100% coverage. You can see some explanations in stackoverflow. Looks byte code has more cases.